src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/README.md

```diff
@@ -0,0 +1,30 @@
+# Elasticsearch Client Instrumentation for OpenTelemetry .NET
+
+Automatically instruments events emitted by the Elasticsearch.Net client library.
```

Comment:
Could you link to the Github page/nuget page for the library? To avoid confusion as I believe there could be multiple ES clients in .net...

Reply:
I added links at the bottom of the readme to the NEST client documentation site. I had links at the top, but markdown lint was complaining about the line being too long.

Reply:
to confirm my understanding:
There are 2 .NET clients in ES. One is NEST (high level) which internally uses Elasticsearch.Net (lowlevel)., which internally uses httpclient.
Elasticsearch.Net is the one that is emitting the diagnosticsource events? So if a user uses ElasticSearch.net or Nest, this package will collect traces from them.

Reply:
> I added links at the bottom of the readme to the NEST client documentation site. I had links at the top, but markdown lint was complaining about the line being too long.

https://marketplace.visualstudio.com/items?itemName=stkb.rewrap - I install this VSCode plugin and do ALT + Q, which does the job for me :)

Reply:
@cijothomas that is correct.

Reply:
Thanks for clarifying. I'd love to have this information in this readme file itself, so users will have no confusion about this. Also  a comment on the SupressInstrumentation, to say that its is the underlying httpclient instrumentation, which is being suppresed. :)

Reply:
Can it take a dependency on just Elasticsearch.Net or does it need NEST + Elasticsearch.Net to work?

Reply:
I just looked at all the changes. Seems like NEST is only referenced by the test project, which is totally fine.

Reply:
I updated the readme and adding HttpClient to the SupressInstrumentation option.